lib/src/kbasesearchengine/main/IndexerWorker.java

```diff
@@ -3,16 +3,7 @@
 import java.io.File;
 import java.io.IOException;
 import java.time.Instant;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
-import java.util.LinkedHashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Pretty minor, but I don't like polluting the namespace with unnecessary names. It also makes the code somewhat harder to read if you're trying to figure out where a name came from and you're using a dumb text editor.

Reply:
That's my intellij editor trying to be intelligent. I reverted.